bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/BootstrapLink.java

```diff
@@ -91,7 +91,7 @@ public BootstrapLink(final String id, final IModel<T> model, final Buttons.Type
 
         add(icon = newIcon("icon"));
         add(splitter = newSplitter("splitter"));
-        add(label = newLabel("label"));
+        add(label = newLabel("label", model));
```

Comment:
Thank you. I'm using wicket for years and I was not aware about the bodyModel in the AbstracLink!
So BootstrapLink definitely need to be refactor. Does it really need a component as a label?